components/guided_experience.js

```diff
@@ -30,49 +29,56 @@ const BlueBar = styled("div")`
   margin-bottom: 40px;
 `;
 
-const styles = () => ({
-  root: {
-    border: "solid 1px grey",
-    backgroundColor: "white",
-    marginTop: "5px",
-    margin: "15px"
-  },
-  box: {
-    padding: "20px"
-  },
-  container: {
-    margin: "0 auto",
-    maxWidth: "1200px",
-    paddingLeft: "16px",
-    paddingRight: "16px"
-  },
-  prevButton: {
-    marginTop: "20px",
-    marginLeft: "15px"
-  },
-  nextButton: {
-    backgroundColor: "black",
-    color: "white",
-    marginTop: 0,
-    margin: "25px",
-    "&:hover": {
-      background: "#606060"
-    }
-  },
-  title: {
-    fontSize: "1.5em",
-    color: "black"
-  },
-  subTitle: {
-    fontSize: "1em"
-  },
-  jumpButton: {
-    fontSize: "1.5em"
-  },
-  comma: {
-    marginRight: "0.5em"
+const root = css`
+  border: solid 1px grey;
+  background-color: white;
+  margin: 15px;
+`;
```

Comment:
no margin-top in here?

Reply:
`margin: "15px"`

is the same as:

```javascript
marginTop:` "15px",
marginRight: "15px",
marginBottom: "15px",
marginLeft: "15px",
```

So `marginTop: "5px"` was getting overwritten before and not doing anything.